kernel-rs/src/virtio/mod.rs

```diff
@@ -221,8 +222,14 @@ bitflags! {
     }
 }
 
-/// This many virtio descriptors. It must be a power of two.
-const NUM: usize = 1 << 3;
+// This many virtio descriptors. It must be a power of two.
+cfg_if! {
+    if #[cfg(feature = "lfs")] {
+        const NUM: usize = 1 << 5;  // LFS requires additional descriptors.
```

Comment:
Could you explain the reason for this?

Reply:
We want to commit 10 disk blocks at once, so 8 descriptors aren't enough. We would need at least 12 in this implementation, or at least 32 if we want to follow [this](https://github.com/kaist-cp/rv6/pull/616#pullrequestreview-918506412) implementation.

Reply:
I don't think we can write all the 30 descriptor chains for individual blocks in the segment to the `avail` ring before incrementing `avail.idx` by the number of blocks written, considering that there may be other processes writing to the `avail` ring.

Reply:
@reverofx We have the lock on the `Disk`, so I think we can write 30 descriptors if we wait enough.

Reply:
Understood.